### ISSUE TITLE
Going all lowercase (via config)?

### ISSUE THREAD

**Issue by thierryk** [yahoo/atomizer#86]
We know why we have chosen to capitalize Atomic classes (for a very good reason), but it seems many people are bothered by this. I'm wondering if we should allow users to go full lowercase... Even though *it is not a smart move*.

**Comment by 3den** [comment]
I like the Capitalized style coz it is visualy diferent from non-atomic classes

**Comment by thierryk** [comment]
Yes, it's our "signature" kinda but it is also a cheap way to namespace things.
This would be an option for people who *cannot* deal with capitalized classes.

**Comment by 3den** [comment]
That would be an optional config, something like: 
```
{
    "classStyle": "lowercase"
}
```

The issue of this config is that if for instance one project uses this it will be incompatible with an other that does not, or we will have to generate duplicated styles...

**Comment by renatoi** [comment]
We can address this issue later if it proves to be a problem for people even though we don't believe it's a smart move. I'm closing this for now.

**Comment by ming-codes** [comment]
> Yes, it's our "signature" kinda but it is also a cheap way to namespace things.
This would be an option for people who cannot deal with capitalized classes.

Here's another idea. [CSS class name can actually be any characters](https://mathiasbynens.be/demo/crazy-class) So taking a page from jQuery, we can prefix atomic class with `@` character as a way for namespacing.

**Comment by renatoi** [comment]
We thought about doing something like this initially, but making it just a capital letter seemed simpler because it requires fewer keystrokes, and would produce a smaller CSS output.

**Comment by thierryk** [comment]
@ming-codes I originally opened this issue because some people complained of our special **casing** but *"namespacing"* should not really be a concern because of the `()` used in ACSS classes. 
As a side note, as suggested by @renatoi, prefixing using special characters could be "expensive" as those would also need to be escaped.

**Comment by matthewmueller** [comment]
@thierryk can you catch us up on the reasoning behind the capital letter for those of us taken aback by it on first sight :-)

**Comment by thierryk** [comment]
@matthewmueller our first implementation of a Atomic CSS architecture at Yahoo! was a static styles sheet meant to be used by various teams across different projects. We needed to prevent name collision but we didn't want to namespace our classes by prefixing them. Instead, we chose to do that by capitalizing them (CSS is case sensitive)&mdash;as we thought nobody was doing that. 

Once we came up with ACSS (Atomic CSS via Atomizer) the capitalisation was not really necessary anymore due to our use of `()` which made our classes kinda *unique*; but we thought that that capital letter would help authors make a clear distinction between ACSS classes and others (like @3den suggested in a previous comment). If I recall, @renatoi mentioned that capitalisation was forcing authors to use the *shift* key more than necessary&mdash;which is a valid concern, but at the end of the day we didn't change anything and kept the capitalization.
